data/data/agent/systemd/units/agent-add-node.service

```diff
@@ -6,7 +6,7 @@ Conflicts=start-cluster-installation.service
 PartOf=assisted-service-pod.service
 After=network-online.target apply-host-config.service
 ConditionPathExists=/etc/assisted/node0
-ConditionPathExists=/etc/assisted/add-nodes.env
+ConditionPathExists=/etc/assisted/workflow/add-nodes.env
```

Comment:
can `workflow` be named as `day2` for better readability?
`ConditionPathExists=/etc/assisted/day2/add-nodes.env`

Reply:
The idea was that it's the `add-nodes.env` file that tells us it's for Day 2.
Since it has to go in a directory, I named the directory `workflow` so that in the future we could potentially add env files for other workflows in there as well.